content/faq/admin/general-faqs.md

```diff
@@ -6,6 +6,7 @@ redirect:
 - /docker-hub/onboarding-faqs/
```

Comment:
**Note to reviewer:** is this line still needed or can it be removed?

Reply:
you can remove the `redirect` key, it doesn't do anything